### PR TITLE
fix: gh fetch error hidden

### PR DESCRIPTION
**Pull request by TomerFi**
@bennypowers 

fixed the exception suppression we discussed.

in regards to the CODEOWNERS file,
I'm of the fence, please share your take on this.

as it's designed so far, you and/or me are listed as owners for all the core stuff.
the _about_, _community_, and _posts_ folders, has no owners.

I can modify this of course, this is just a suggestion.
WDYT?

### CODE REVIEW

File: .github/CODEOWNERS
Comment (bennypowers) [inline .github/CODEOWNERS:5]: you'll get a ping for all the dot-files and such

File: .github/CODEOWNERS
Comment (bennypowers) [inline .github/CODEOWNERS:7]: you'll get pinged on layout and templates stuff

File: _data/contributors.js
Comment (bennypowers) [inline _data/contributors.js:72]: thanks for 'catch'ing this :drum:

**Review by bennypowers — APPROVED** [review]

**Comment by bennypowers** [comment]
LGTM if you're happy with the CODEOWNERS setup, resolve above notes to merge

File: .github/CODEOWNERS
Comment (TomerFi) [reply .github/CODEOWNERS:5]: only for ones on the root (including root folders).
these are mainly core files no?

File: .github/CODEOWNERS
Comment (TomerFi) [reply .github/CODEOWNERS:7]: yes, are they changing frequently?
I figured, once we get rolling, most changes would be in the posts folder no?

Comment (TomerFi) [reply .github/CODEOWNERS:7]: you know what... let's keep it like this for now.
I'll remove myself if it get out of hand.

**Pull request merged**
